content/en/agent/guide/network.md

```diff
@@ -145,6 +145,19 @@ Open the following ports in order to benefit from all the Agent functionalities:
 
 For a detailed configuration guide on proxy setup, see [Agent Proxy Configuration][8].
 
+## Agent-side data buffering on network unavailability
+
+When the network becomes unavailable, the Agent stores the traffic in the memory.
+The maximum memory usage for storing the traffic is defined by the configuration `forwarder_retry_queue_payloads_max_size`. When this limit is reached, the traffic is dropped.
+
+The Agent `7.27.0` and above can store the traffic on the disk when the memory limit is reached.
+This feature is enabled by setting `forwarder_storage_max_size_in_bytes` to a positive value.
+`forwarder_storage_max_size_in_bytes` defines the maximum amount of storage space the Agent can use to store the traffic on the disk.
+
+The traffic is stored in the folder defined by `forwarder_storage_path` which is by default `/opt/datadog-agent/run/transactions_to_retry` on Unix systems and `C:\ProgramData\Datadog\run\transactions_to_retry` on Windows.
+
+Note: In order to avoid running out of storage space, the Agent can store the traffic on the disk only if the total storage space used is less than 95%. This limit is defined by `forwarder_storage_max_disk_ratio`.
```

Comment:
```suggestion
To avoid running out of storage space, the Agent stores traffic on disk only if the total storage space used is less than 95 percent. This limit is defined by `forwarder_storage_max_disk_ratio` setting.
```